plugins/modules/ec2_vpc_dhcp_option.py

```diff
@@ -130,39 +130,86 @@
     type: str
     returned: when available
 dhcp_config:
```

Comment:
I would probably understand if we ned to return anymore boto2-style data. Maybe, we do not need it anymore and we can deprecate directly dhcp_config.

@tremble thoughts?

Reply:
+1 to removing the duplication and old boto2 values. 

You've got the history about right, when we first migrated things over from boto2 to boto3 it was a very slow process so we pretty much had to keep things like this to avoid constant churn.  This is also why some of the modules with older heritage have parameters that don't line up with the boto3 naming conventions.

If we're going to clean up things like this we should probably try to do so in a batch, just to avoid churn.  It's frustrating to need to constantly revisit playbooks because of deprecations.